packages/python-packages/apiview-stub-generator/tests/_test_util.py

```diff
@@ -10,26 +10,37 @@
 
 def _tokenize(node):
     apiview = ApiView(pkg_name="test", namespace="test")
-    apiview.tokens = []
-    node.generate_tokens(apiview)
-    return apiview.tokens
+    node.generate_tokens(apiview.review_lines)
+    return apiview.review_lines
 
 
-""" Returns the tokens rendered into distinct lines. """
-def _render_lines(tokens) -> List[str]:
-    return "".join([x.render() for x in tokens]).splitlines()
+""" Returns the review line tokens rendered into distinct lines. """
 
 
-""" Returns the tokens as a single concatenated string. """
-def _render_string(tokens) -> str:
-    lines = "".join([x.render() for x in tokens]).splitlines()
+def _render_lines(review_lines) -> List[str]:
+    lines = review_lines.render()
+    return [x.lstrip() for x in lines]
```

Comment:
I had similar logic for stripping leading or trailing whitespace in Swift and TypeSpec and getting rid of it actually made my tests and the resulting code better. So you might try removing this lstrip to see the impact.